application/templates/pages/about/performance.md

```diff
@@ -0,0 +1,35 @@
+Performance indicators showing how the planning data platform is making it easier to find, use and trust planning and housing data in England.
+
+Last updated 19 July 2024.
+
+## Availability
+
+### 89 datasets on the platform
+
+**00 national datasets** available from **00 organisations**. 0 added last month (+/-0%).
+
+**00 datasets** provided by **00 local planning authorities**. 0 provided last month (+/-0%).
+
+0 new authorities providing data. +/-0% change from last week.
+
+## Quality
+
+### 00 out of 00 quality score
+
+00 datasets with no issues (0%). 00 fixed last week (+/-0%).
+
+00 datasets conform to the specifications (0%). 00 improved last week (+/-0%).
+
+00 datasets up to date (0%). 00 updated last week (+/-0%).
+
+## Coverage
```

Comment:
Would this exclude the national datasets and refer only to LPA endpoints, as coverage for a national dataset is already 100%?

Reply:
Good question! I think we should include national datasets even though we expect the coverage to be 100% because it's a more truthful picture. Data provided by LPAs contributes to national coverage, and perhaps we can have a separate subset metric showing coverage from LPAs as a percentage of what we expect (like you've suggested in another comment).